### PR TITLE
fix: 4496 linea bridge changed

### PR DESCRIPTION
**Pull request by liu-zhipeng**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

**Review by rhlsthrm — APPROVED** [review]

File: packages/agents/lighthouse/src/tasks/propagate/helpers/consensys.ts [hunk 1]
```diff
@@ -19,7 +19,8 @@ export const getPropagateParams = async (
     l2ChainId,
   });
 
-  const _fee = utils.parseEther("0.01").toString();
+  // the additional optional "postman" fee = 0 currently
+  const _fee = utils.parseEther("0").toString();
```
Comment (LayneHaber) [inline packages/agents/lighthouse/src/tasks/propagate/helpers/consensys.ts:23]: is there a way to pull this dynamically? can you provide a link to where this is documented?

File: packages/agents/lighthouse/src/tasks/sendOutboundRoot/helpers/consensys.ts [hunk 2]
```diff
@@ -3,7 +3,9 @@ import { utils } from "ethers";
 import { ExtraSendOutboundRootParam } from "../operations/sendOutboundRoot";
 
 export const getSendOutboundRootParams = async (_l2domain: string): Promise<ExtraSendOutboundRootParam> => {
-  const _fee = utils.parseEther("0.01").toString();
+  // the mandatory L2->L1 anti-DDoS (or service) fee, currently fixed at 0.001
+  // the additional optional "postman" fee = 0 currently
```
Comment (LayneHaber) [inline packages/agents/lighthouse/src/tasks/sendOutboundRoot/helpers/consensys.ts:7]: link to documentation?

**Review by LayneHaber — COMMENTED** [review]
how has this been tested post deployment? do we need to update staging deployments + subgraphs as well?

File: packages/agents/lighthouse/src/tasks/propagate/helpers/consensys.ts (hunk 1 above)
Comment (liu-zhipeng) [reply packages/agents/lighthouse/src/tasks/propagate/helpers/consensys.ts:23]: Linea team said we can estimate `postman` fee using their SDK, but SDK is not available yet. I couldn't find any doc for that. just from the direct chat with them

File: packages/agents/lighthouse/src/tasks/sendOutboundRoot/helpers/consensys.ts (hunk 2 above)
Comment (liu-zhipeng) [reply packages/agents/lighthouse/src/tasks/sendOutboundRoot/helpers/consensys.ts:7]: ^^ Same

**Comment by rhlsthrm** [comment]
> how has this been tested post deployment? do we need to update staging deployments + subgraphs as well?

We don't have staging Linea deployment.

File: packages/agents/lighthouse/src/tasks/propagate/helpers/consensys.ts (hunk 1 above)
Comment (rhlsthrm) [reply packages/agents/lighthouse/src/tasks/propagate/helpers/consensys.ts:23]: We can leave this for now and know we have to change it later once the SDK is available.

**Review by rhlsthrm — APPROVED** [review]

**Review by wanglonghong — APPROVED** [review]

**Pull request merged**